src/tools/run-make-support/src/env_checked.rs

```diff
@@ -0,0 +1,20 @@
+use std::ffi::OsString;
```

Comment:
Why the name `checked`? We do check/panic/assert pretty much everything in `run-make-support`, but the other modules are not named like this.

Reply:
Naming is hard. Renamed to `env`.